engine/schema/migrations/20171006131622_drop_groups_entity_table.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class DropGroupsEntityTable extends AbstractMigration
+{
+    /**
+     * Move groups_entity attributes to metadata
+     */
+    public function up() {
+
+		if (!$this->hasTable('groups_entity') || !$this->hasTable('metadata')) {
+			return;
+		}
+
+		$prefix = $this->getAdapter()->getOption('table_prefix');
+		$rows = $this->fetchAll("
```

Comment:
I am not sure how this is going to scale. I don't think Phinx uses iterator, and on large sites this might just run out of memory. You should be able to add a helper class and use ElggBatch without a callback, I think.

Reply:
On a second thought better not to rely on core classes, as they might not be available yet.

Reply:
we could do a while loop fetching 1 or 25 per loop